server/package.json

```diff
@@ -50,6 +50,9 @@
     "@nestjs/typeorm": "^10.0.0",
     "@paljs/plugins": "^5.3.3",
     "@prisma/client": "4.13.0",
+    "@ptc-org/nestjs-query-core": "^4.2.0",
```

Comment:
This package is a fork of nestjs-query-core which seems to be well maintained compared to the original one

Reply:
Yes it's the one to go with !